arkiverer/src/test/kotlin/no/nav/soknad/arkivering/soknadsarkiverer/service/TaskListServiceTests.kt

```diff
@@ -12,21 +12,53 @@ import no.nav.soknad.arkivering.soknadsarkiverer.utils.createSoknadarkivschema
 import no.nav.soknad.arkivering.soknadsarkiverer.utils.loopAndVerify
 import org.junit.jupiter.api.AfterEach
 import org.junit.jupiter.api.Assertions.*
+import org.junit.jupiter.api.BeforeAll
 import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.TestInstance
+import org.junit.jupiter.api.extension.ExtendWith
 import org.mockito.ArgumentCaptor
+import org.mockito.Mock
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.beans.factory.annotation.Value
+import org.springframework.boot.context.properties.ConfigurationProperties
+import org.springframework.boot.context.properties.EnableConfigurationProperties
+import org.springframework.boot.test.context.ConfigDataApplicationContextInitializer
+import org.springframework.boot.test.context.TestConfiguration
+import org.springframework.boot.test.mock.mockito.MockBean
+import org.springframework.context.annotation.Bean
+import org.springframework.context.annotation.Import
+import org.springframework.context.annotation.PropertySource
+import org.springframework.test.annotation.DirtiesContext
+import org.springframework.test.context.ActiveProfiles
+import org.springframework.test.context.ContextConfiguration
+import org.springframework.test.context.TestPropertySource
+import org.springframework.test.context.junit.jupiter.SpringExtension
 import java.util.*
 import java.util.concurrent.TimeUnit
 
+@ExtendWith(SpringExtension::class)
+@ContextConfiguration(initializers = [ConfigDataApplicationContextInitializer::class])
+@ActiveProfiles("test")
+@Import(*[TaskListConfig::class,AppConfiguration::class,MetricsTestConfig::class])
```

Comment:
The spread operator is a bit weird. I'd recommend changing this line to
`@Import(value = [TaskListConfig::class, AppConfiguration::class, MetricsTestConfig::class])`

Reply:
Kan ikke gjøre det. Compiler feil.

Reply:
Hvis du kun fjerner `*` så blir det kompileringsfeil. Men hvis du legger på `value = ` så virker det. Because of course.